src/genie/libs/parser/iosxe/tests/ShowCtsRoleBasedPermissions/cli/equal/sample_output_expected.py

```diff
@@ -0,0 +1,10 @@
+expected_output = {
+    "indexes": {
```

Comment:
based on the change, below can be possible, right?
```
expected_output = {
    "indexes": {}
}
```
could you please add the output as unittest?